middleware/grpc_stats.go

```diff
@@ -31,6 +31,7 @@ type contextKey int
 
 const (
 	contextKeyMethodName contextKey = 1
+	contextKeyRouteName  contextKey = 2
```

Comment:
nit: move to route_injector.go?

Reply:
I wanted to keep all the `contextKey` values in one place, as otherwise there's a risk that we define two `contextKey`s with the same value (eg. two `contextKey`s with value `3`).